index.html

```diff
@@ -38,6 +37,28 @@ <h2>Let's Connect</h2>
                 <li>GitHub</li>
             </ul>
       </section>
+      
+      <section id="messageForm">
+        <h2>Leave a Message</h2>
+        <form name="leave_message">
+          <label for="name">Name:</label><br>
+          <input type="text" name="name" required = "true"><br>
```

Comment:
```suggestion
          <input type="text" name="name" required="true"><br>
```
Not a big deal: IDEs can do formatting clean-ups like this.